### PR TITLE
refactor: remove load_as_component and update doc

### PR DESCRIPTION
**Pull request by elliotzh**
# Description

Remove previous interface `load_as_component` and update corresponding tests and documents.

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes].**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [x] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: docs/how-to-guides/init-and-test-a-flow.md [hunk 1]
```diff
@@ -294,4 +294,5 @@ Break points and debugging functionalities for the Python steps in your flow. Ju
 
 ## Next steps
 
-- [Run and evaluate a flow](./run-and-evaluate-a-flow.md)
\ No newline at end of file
+- [Run and evaluate a flow](./run-and-evaluate-a-flow.md)
+- [Use Flow in Pipeline Job](./use-flow-in-pipeline.md)
```
Comment (wangchao1230) [inline docs/how-to-guides/init-and-test-a-flow.md:298]: only cap the first word

File: docs/how-to-guides/use-flow-in-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Use Flow in Pipeline Job
```
Comment (wangchao1230) [inline docs/how-to-guides/use-flow-in-pipeline.md:1]: only cap the first word

File: examples/flows/standard/web_classification/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+# Web Classification
```
Comment (wangchao1230) [inline examples/flows/standard/web_classification/README.md:1]: why do you need to add a new flow example folder?

File: examples/tutorials/flow-in-pipeline/pipeline.ipynb [hunk 4]
```diff
@@ -0,0 +1,220 @@
+{
+ "cells": [
```
Comment (wangchao1230) [inline examples/tutorials/flow-in-pipeline/pipeline.ipynb:2]: add github ci for this example

File: examples/flows/standard/web_classification/README.md (hunk 3 above)
Comment (elliotzh) [reply examples/flows/standard/web_classification/README.md:1]: flow in pipeline will use the name of flow directory as component name, but `web-classification` is not a valid component name. Not sure if we should do a automatic transformation in `azure-ai-ml`

Comment (elliotzh) [reply examples/flows/standard/web_classification/README.md:1]: or let me always set name in current example

**Review by zhengfeiwang — DISMISSED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Review by 0mza987 — APPROVED** [review]

**Pull request merged**